modules/default/calendar/calendar.js

```diff
@@ -237,18 +237,6 @@ Module.register("calendar", {
 				symbolWrapper.className = "symbol align-right " + symbolClass;
 
 				const symbols = this.symbolsForEvent(event);
-				// If symbols are displayed and custom symbol is set, replace event symbol
-				if (this.config.displaySymbol && this.config.customEvents.length > 0) {
```

Comment:
This check isnt in the added code. Dont we need it there?

Reply:
I don't believe so, the check was redundant in the old location, it was already inside an if (this.config.displaySymbol), and the other call should have the custom symbol information even if symbols aren't being displayed, since it's populating the notification message